### DISCUSSION TITLE
Proposta di nuovi allegati per casi d'uso di Matrimonio.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#932] [Miglioramenti] [answered]
Buongiorno,
 
riportiamo una segnalazione dal Comune di Carpi:
 
il fascicolo dei matrimoni si compone dei documenti acquisiti durante il processo di pubblicazione a cui possono aggiungersi nuovi allegati al momento della compilazione dell’atto di matrimonio.
Per rendere più completo il fascicolo si propone di aggiungere, come facoltativi, i seguenti allegati:
 
- 'Nullaosta di cui all'art. 116 o documentazione equivalente' (già censito con id: 103);
- 'Verbale di pubblicazione' (nuovo da censire).
 
Ai seguenti casi d'uso:
311111 - Matrimonio con Rito Civile nella casa comunale;
311121 - Matrimonio con Rito Civile nella casa comunale con Riconoscimento di uno o più figli;
311211 - Matrimonio con Rito Civile fuori dalla casa comunale;
311221 - Matrimonio con Rito Civile fuori dalla casa comunale con Riconoscimento di uno o più figli;
321311 - Matrimonio con Rito Religioso Cattolico;
321321 - Matrimonio con Rito Religioso Cattolico con Riconoscimento di uno o più figli;
321411 - Matrimonio con Rito Religioso Acattolico;
321421 - Matrimonio con Rito Religioso Acattolico con Riconoscimento di uno o più figli.
 
 
 
In attesa di riscontro,
 
cordiali saluti.

**Comment by mttfranci** [comment, marked as answer]
Grazie per il suggerimento.
Da notare che è possibile aggiungere un numero qualsivoglia di allegati aggiuntivi di tipologia "Allegato generico" (998).
Comunque valuteremo, anche con il gruppo di analisi, se è oppurtuno inserire la vostra richiesta nella pianificazione.

Saluti

Assistenza ANSC

**Comment by mttfranci** [comment]
@gcuci per quanto riguarda "'Verbale di pubblicazione' (nuovo da censire)." ritenete sia diverso dall'allegato già censito : 
87 - Atto di pubblicazione

Grazie in anticipo.

**Comment by gcuci** [comment]
Si, i due documenti sono differenti; il primo, il verbale di pubblicazione, è il documento che viene sottoscritto dagli sposi alla data della pubblicazione e che attualmente rimane agli atti all'interno del fascicolo di matrimonio; il secondo, l'atto di pubblicazione, è il documento che indica i dati degli sposi, il rito e il luogo di celebrazione, su cui viene apposto il bollo, che viene pubblicato online per 8 gg per dare pubblicità a terzi.

**Comment by asqueglia** [comment]
Buonasera con la versione 1.32 è stata corretta la segnalazione. Chiudiamo pertanto la issue
